en/docs/learn/logging-in-to-salesforce-with-facebook.md

```diff
@@ -21,15 +21,13 @@ exits in Facebook, the user is allowed to log in to Salesforce.
 			**make sure to configure it before you begin working with WSO2 IS**.
         
 
-		1.  Open the \<
-			`              PRODUCT_HOME>/repository/conf/carbon.xml             `
+		1.  Open the `              <IS_HOME>/repository/conf/deployment.toml             `
 			file.
-		2.  Look for the commented out configuration
-			`               EnableEmailUserName              ` . Uncomment the
-			configuration to enable email authentication.
+		2.  Add  `               enable_email_domain              ` as shown below.
 
 			``` html/xml
-			<EnableEmailUserName>true</EnableEmailUserName>
+            [tenant_mgt]
+            enable_email_domain= true
 			```
```

Comment:
indent these two properties to come right under the ``` html/xml. Otherwise it will not appear properly on the site

Reply:
corrected.